### DISCUSSION TITLE
Error when loading the file generated by Dream3D-NX

### DISCUSSION THREAD

**Discussion by 123fgh123** [damask-multiphysics/DAMASK#53] [Q&A]
I tried to load .dream3d file generated by newest version Dream3D, that is, Dream3D-NX. But it would lead to an error.

```python
import damask
import numpy as np

grid= damask.GeomGrid(np.zeros([128,1,128],int),[128,1,128]).load_DREAM3D("Dream3D-NX-RVE.dream3d")

ValueError: could not determine base group in file "<HDF5 file "Dream3D-NX-RVE.dream3d" (mode r)>"
``` 

I have attached the .dream3d for further check.
[Dream3D-NX-RVE.zip](https://github.com/user-attachments/files/18240303/Dream3D-NX-RVE.zip)

**Comment by MarDiehl** [comment]
I can confirm that it is currently not possible to load version 8.0 of the DREAM3D file format.
